src/components/DopplerPlus/PlanCalculator.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react';
+import { Slider } from '../shared/Slider/Slider';
+
+const PlanCalculator = () => {
+  // TODO: get data from a double service
+  const [state, setState] = useState({ price: 15 });
+  const valuesPerRange = [15, 29, 48, 77, 106, 145, 240, 340, 460];
```

Comment:
How are you planning to deal with proportionality? or it is not important?

Reply:
It's not important. The important here is values amount that we should mange in the slider.

Reply:
But it will, or I am missunderstanding something?

Reply:
All the values in the Slider will have the same space between them.